rust/lib/src/lib.rs

```diff
@@ -168,7 +168,7 @@ pub fn init_from_b64(
         Ok((c, h)) => (config, _lightwalletd_uri) = (c, h),
         Err(s) => return s,
     }
-    let decoded_bytes = match decode(&base64_data) {
+    let decoded_bytes = match STANDARD_NO_PAD.decode(&base64_data) {
```

Comment:
We need to be sure that the App can open the existent file, and the existent file have some kind of `padding`. Maybe is a good idea that trying to open with `padding` and if fails with `no padding`... to keep backwards compatibility.

Reply:
im implementing this now

Reply:
Let's add some unit-tests of obvious cases.  Maybe proptest.